plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/LabelImageLoader.java

```diff
@@ -42,7 +42,8 @@
    *
    * Must be called in the UI context.
    */
-  void loadImage(String imageUrl, Label label) throws MalformedURLException {
+  @VisibleForTesting
+  public void loadImage(String imageUrl, Label label) throws MalformedURLException {
```

Comment:
do you need to make this public? It looks like the test is in the same package.

Reply:
Mockito couldn't mock non-public method, and that was why I experienced the weird test failure that confused me.

Reply:
Strange. Mockito should be able to mock a non-final package protected method.

Reply:
Yeah, I also think this is very weird. A non-plugin JUnit run worked, but it calls the actual method in the plugin JUnit run. Then I guessed maybe it's because the method is not public. I don't have any insights why it behaves differently between plugin and non-plugin runs.

Reply:
so what's failing when this isn't public? Is that a Travis run or just a pluign test in Eclipse?

Reply:
Both a local plugin test in Eclipse and Travis fail. I guess `mvn verify` runs `.login.ui.test` in the plugin test mode.